vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -498,4 +498,157 @@ def _load_fp8_scale(self, param: torch.nn.Parameter,
                 param_data[expert_id][idx] = loaded_weight
             # If we are in the row parallel case (down_proj)
             else:
-                param_data[expert_id] = loaded_weight
\ No newline at end of file
+                param_data[expert_id] = loaded_weight
+
+
+class GPTQFusedMoE(torch.nn.Module):
```

Comment:
I think `GPTQMarlinMoEMethod` looks good. We'd want to use `FusedMoE` as opposed to introducing `GPTQFusedMoE` - we can work on this tomorrow in making this happen.